libraries/AP_ADSB/AP_ADSB.h

```diff
@@ -138,6 +150,18 @@ class AP_ADSB {
     // confirm a value is a valid callsign
     static bool is_valid_callsign(uint16_t octal) WARN_IF_UNUSED;
 
+    bool ident_is_active() const {
```

Comment:
It would be nice to get some comments above these new methods to clarify what they are for.  Same goes for the new members, "ident_pending" and "ident_isActive".

Reply:
renamed and added comment and context